libs/cardano-protocol-tpraos/src/Cardano/Protocol/TPraos/API.hs

```diff
@@ -186,6 +192,36 @@ instance CC.Crypto c => GetLedgerView (MaryEra c)
 
 instance CC.Crypto c => GetLedgerView (AlonzoEra c)
 
+-- Note that although we do not use TPraos in the Babbage era, we include this
+-- because it makes it simpler to get the ledger view for Praos.
+instance CC.Crypto c => GetLedgerView (BabbageEra c) where
+  currentLedgerView
+    NewEpochState
+      { nesPd,
+        nesEs
+      } =
+      LedgerView
+        { lvD = getField @"_d" . esPp $ nesEs,
+          lvExtraEntropy = error "Extra entropy is not set in the Babbage era",
```

Comment:
maybe it would be less scary to use the neutral nonce? though perhaps failing hard is better? :thinking:

Reply:
I'm inclined to go for failing hard, though I could be argued the other way!

Reply:
Let's fail hard!

```
Do not go gentle into that good night,
Old age should burn and rave at close of day;
Rage, rage against the dying of the light.
```